src/client/app/material.module.ts

```diff
@@ -0,0 +1,74 @@
+import { OverlayModule } from '@angular/cdk/overlay';
```

Comment:
I've been thinking if we need this material module? should we put this in shared? I am not sure though or maybe just put this inside "material" folder, what do you think?

Reply:
Just keep it inside the material module. Doesn't need to be extracted from there. It's fine